client/telemetry/src/layer.rs

```diff
@@ -60,20 +60,20 @@ where
 					let mut vis = TelemetryAttrsVisitor(&mut attrs);
 					event.record(&mut vis);
 
-					match attrs {
-						TelemetryAttrs {
-							message_verbosity: Some(message_verbosity),
-							json: Some(json),
-							..
-						} => {
-							let _ = sender.try_send((
-								message_verbosity
-									.try_into()
-									.expect("telemetry log message verbosity are u8; qed"),
-								json.clone(),
-							));
-						}
-						_ => panic!("missing fields in telemetry log: {:?}", event),
+					if let TelemetryAttrs {
+						verbosity: Some(verbosity),
+						json: Some(json),
+						..
+					} = attrs
+					{
+						let _ = sender.try_send((
+							verbosity
+								.try_into()
+								.expect("telemetry log message verbosity are u8; qed"),
+							json,
+						));
+					} else {
+						panic!("missing fields in telemetry log: {:?}", event);
```

Comment:
```suggestion
                        // This can happen if `tracing::info_span!` is (mis-)used with the telemetry target directly; users should use the `telemetry!` macro.
						panic!("missing fields in telemetry log: {:?}", event);
```

…or something to that effect?

Reply:
I will copy the message into the panic. There is no reason for it to be outside the panic message

Reply:
Done in 1fe185b17